GOVERNANCE.md

```diff
@@ -0,0 +1,191 @@
+
+* "Responsibilities" – functions of a member 
+* "Requirements" –  qualifications of a member 
+* "Privileges" –  entitlements of member
+
+
+### Contributor
+Description: A Contributor contributes directly to the project and adds value to it. Contributions need not be code. People at the Contributor level may be new contributors, or they may only contribute occasionally.
+
+* Responsibilities include:
+    * Follow the CNCF CoC
+    * Follow the project contributing guide
+* Requirements (one or several of the below):
+    * Report and sometimes resolve issues against any of the project’s repositories
+    * Occasionally submit PRs against any of the project’s repositories
+    * Contribute to project documentation, including the manpages, tutorials, and Podman.io
+    * Attend community meetings when reasonable
+    * Answer questions from other community members on the mailing list, Slack, Matrix, and other communication channels
+    * Assist in triaging issues
+    * Assist in reviewing pull requests, including testing patches when applicable
+    * Test release candidates and provide feedback
+    * Promote the project in public
+    * Help run the project infrastructure
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become a Reviewer
+
+### Reviewer
+Description: A Reviewer has responsibility for the triage of issues and review of pull requests on the Podman project or a subproject, consisting of one or more of the Git repositories that form the project. They are collectively responsible, with other Reviewers, for reviewing changes to the repository or repositories and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.
+
+Reviewers have all the rights and responsibilities of an Contributor, plus:
+
+* Responsibilities include:
+    * Regular contribution of pull requests to the Podman project or its subprojects
+    * Triage of Github issues on the Podman project or its subprojects
+    * Regularly fixing Github issues on the Podman project or its subprojects 
+    * Following the reviewing guide
+    * A sustained high level of pull request reviews on the Podman project or one of its subprojects
+    * Assisting new Contributors with the process of contributing
+    * Helping other contributors become reviewers
+* Requirements:
+    * Has a proven record of good-faith contributions to the project as a Contributor for a period of at least 6 months. The time requirement may be overridden by a supermajority vote of Maintainers.
+    * Has participated in pull request review and/or issue triage on the project for at least 6 months. The time requirement may be overridden by a supermajority vote of Maintainers.
+    * Is supportive of new and occasional contributors and helps get useful PRs in shape to merge
+* Additional privileges:
+    * Has rights to approve pull requests in the Podman project or a subproject
```

Comment:
I don't think it is defined what approving means? It is not clear to me what it means, is it simply to say it has been reviewed by a reviewer?

Reply:
Github's "approve" functionality, and/or the `/approve` function in the bot